pkg/loader/load.go

```diff
@@ -184,20 +209,28 @@ func NewLoader(db *gosql.DB, opt ...Option) (Loader, error) {
 		o(&opts)
 	}
 
+	log.Info("new loader", zap.String("opts", fmt.Sprintf("%+v", opts)))
+
+	if !opts.enableDispatch {
+		opts.workerCount = 1
+		opts.batchSize = math.MaxInt64
```

Comment:
Why do we change this behavior (to `math.MaxInt64`)?

Reply:
add comment in https://github.com/pingcap/tidb-binlog/pull/915/commits/aeb33cda5c59d96b3c07adf8bc9835c088d1860f